pkgs/development/tools/smlfmt/default.nix

```
@@ -0,0 +1,37 @@
+{ lib, stdenv, fetchFromGitHub, pkg-config, mlton }:
+
+stdenv.mkDerivation rec {
+  pname = "smlfmt";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "shwestrick";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-7CTfMiEvuOena5SOE0qKpFPq6ARxmkE6d+loznZNbC0=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ mlton ];
```

I think it only needs mlton
```suggestion
  nativeBuildInputs = [ mlton ];
```